command/report/report.go

```diff
@@ -52,16 +52,16 @@ func NewCmdReport() *cobra.Command {
 	}
 
 	// --repo, -r flag
-	cmd.Flags().StringVar(&opts.Analyzer, "analyzer", "", "The analyzer shortcode whose test coverage is being reported")
+	cmd.Flags().StringVar(&opts.Analyzer, "analyzer", "", "name of the analyzer (example: test-coverage)")
```

Comment:
The descriptions for `analyzer` and `key` are confusing, users might confuse `analyzer` as the language for which they are uploading the artifact. IMO, we can be more descriptive.

I'll think about and suggest something but in the meantime, if you get a better description, you can update it.